lib/core/localization/l10n/app_vi.arb

```
@@ -151,5 +151,15 @@
   "retry": "Thử lại",
   "noData": "Không có dữ liệu",
-  "guest": "Khách"
+  "guest": "Khách",
+  
+  "failedToLoadHotelDetails": "Không thể tải thông tin khách sạn",
+  "about": "Giới thiệu",
+  "amenities": "Tiện nghi",
+  "nearbyPlaces": "Địa điểm lân cận",
+  "availableRooms": "Phòng có sẵn",
+  "perNight": "mỗi đêm",
+  "guest": "Khách",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)